src/workflow/snakemake/rules/rule_start_benchmark.smk

```diff
@@ -0,0 +1,19 @@
+import os
+
+rule start_benchmark:
```

Comment:
(not urgent but) I think we could get rid of this. Or: better, replace by some archspec/nproc/OS fingerprinting, so we have some extra logging about the hardware used to run snakemake

Reply:
I'll consider this as nice to have for now, would it be better to be included in a larger issue regarding logging?